src/style.css

```diff
@@ -1 +1,8 @@
 /* styles go here */
+.form-field {
+  margin-bottom: 10px;
+}
+
+.groups {
+  margin-bottom: 20px;
+}
```

Comment:
1) apply these classes for each fieldset and form-field
2) and you can use styles like this
```
.form-field:not(:last-child) {
  margin-bottom: 10px;
}
```